src/main/java/mate/academy/hibernate/relations/lib/Factory.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.lib;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.CONSTRUCTOR)
+public @interface Factory {
```

Comment:
These annotations are not needed. But well done for experimenting! :)

Reply:
Do you mean Retention and Target or all annotations including Factory?